xml/admin_ceph_troubleshooting.xml

```diff
@@ -515,4 +515,101 @@ keep_jobs: 1</screen>
    </procedure>
   </tip>
  </sect1>
+ <sect1 xml:id ="ceph-fix-too-many-pgs">
+   <title>Too Many PGs Per OSD</title>
+   <para>
+     The <option>TOO_MANY_PGS</option> flag is raised when the number of PGs in
+     use is above the configurable threshold of <option>mon_pg_warn_max_per_osd</option>
+     PGs per OSD. If this threshold is exceeded, the cluster does not allow new pools
+     to be created, pool <option>pg_num</option> to be increased, or pool
+     replication to be increased.
+   </para>
+   <para>
+     &productname; &prevproductnumber; and &productnumber; have two ways to
+     solve this issue.
+   </para>
+   <procedure>
+     <title>Solution 1</title>
+     <step>
+       <para>
+         Set the following in your <filename>ceph.conf</filename>:
+       </para>
+<screen>
+[global]
+
+mon_max_pg_per_osd = 800  #  depends on your amount of PGs
+osd max pg per osd hard ratio = 10 #  the default is 2. We recommend to set at least 5.
+mon allow pool delete = true # without it you can't remove a pool
```

Comment:
This should NEVER be in a ceph.conf!! We use "ceph tell mon.* injectargs '--mon_allow_pool_delete=true' " to set this temporarily only and unset it after the deletion with "ceph tell mon.* injectargs '--mon_allow_pool_delete=false' "

--> also keep in mind that changing ceph.conf requires mon restart to get it effective and the changes in ceph.conf would require to be done via deepsea..